### ISSUE TITLE
Built-in XServer in windows

### ISSUE THREAD

**Issue by thethereza** [cascadium/wsl-windows-toolbar-launcher#48]
Doesn't windows now have a native xserver built in that can be used?  I actually recall being able to use it, then upgraded to windows 11 (sucked), reinstalled windows 10 - but now I can't seem to get it to work.

**Comment by fquinner** [comment]
Yeah it's an odd one. The native xserver was once on an insiders windows 10 build, but they pulled that functionality to be exclusively windows 11 once that was announced, much to my irritation.
